crate/operator/config.py

```diff
@@ -82,16 +82,16 @@ class Config:
     #: allow for easier testing in smaller Kubernetes clusters.
     TESTING: bool = False
 
-    #: HTTP Basic Auth password for web requests made to :attr:`WEBHOOK_URL`.
```

Comment:
The settings are deliberately sorted this way: alphabetical 😉. Even though username and password belong together, "password" comes before "URL" . Same for the assignment below inside `load()`.

Reply:
Who knew! :) Thanks, I'll revert.